doc/changes/names.inc

```diff
@@ -436,4 +436,6 @@
 
 .. _John Veillette: https://psychology.uchicago.edu/directory/john-veillette
 
-.. _Nikolai Chapochnikov: https://github.com/chapochn
\ No newline at end of file
+.. _Nikolai Chapochnikov: https://github.com/chapochn
+
+.. _Matthias Dold: https://matthiasdold.de
```

Comment:
@matthiasdold your URL has a certificate error:

https://matthiasdold.de/

Let me know if you can fix it, otherwise I'll add it to the list of links we ignore in our `linkcheck` CI run

cc @drammock this was one of our failures this week

Reply:
@larsoner - sorry, certbot hug up on me... it should be fixed now and not occur again.
Thanks for pointing it out though!

Reply:
Thanks for fixing it so quickly!